src/components/Landing/TrendingSection.tsx

```diff
@@ -0,0 +1,59 @@
+import { Row, Col, Button } from 'antd'
+
+import { CSSProperties, useContext } from 'react'
+import { ThemeContext } from 'contexts/themeContext'
+import TrendingProjects from 'components/Projects/TrendingProjects'
+
+export default function TrendingSection() {
+  const { isDarkMode } = useContext(ThemeContext)
+
+  const trendingProjects: CSSProperties = {
+    // Light theme uses a slightly lighter background than background-l1
+    backgroundColor: isDarkMode ? 'var(--background-l1)' : '#e7e3dc80',
+    paddingTop: 40,
```

Comment:
It's just a default react thing, I've always been in the habit of using a number value where possible in case we ever want to operate on them, and personally think it looks nicer 😅

Reply:
cooool cool, TIL. Thanks 🚀 https://react-cn.github.io/react/tips/style-props-value-px.html